### PR TITLE
Added Readme files documentation for Cortx-Provisioner and Provisione…

### PR DESCRIPTION
**Pull request by tanujashinde0405**
…r Test Framework

Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

# Problem Statement
- Added Readme files documentation for Cortx-Provisioner and Provisioner Test Framework

# Design
- Add Documentation for Provisioner framework.

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [x] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [x] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered src/README.md](https://github.com/Seagate/cortx-prvsnr/blob/readme-update/src/README.md)
[View rendered test/deploy/kubernetes/README.md](https://github.com/Seagate/cortx-prvsnr/blob/readme-update/test/deploy/kubernetes/README.md)
[View rendered test/setup/README.md](https://github.com/Seagate/cortx-prvsnr/blob/readme-update/test/setup/README.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by cahuja123 — APPROVED** [review]
LGTM

**Review by kailashyadav11 — APPROVED** [review]
LGTM

File: test/setup/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+yum install --nogpgcheck -y python3 cortx-prereq
+yum install --nogpgcheck -y cortx-py-utils python36-cortx-prvsnr
+yum install --nogpgcheck -y cortx-provisioner
+yum install --nogpgcheck -y cortx-motr cortx-hare cortx-s3server cortx-csm_agent cortx-csm_web
+```
+
+## Testing Manually
+
+Run apply command
+```bash
```
Comment (nitin-seagate) [inline test/setup/README.md:29]: Why do we need bash to execute python exec...

File: test/setup/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+cortx_setup.py config apply -f yaml://`pwd`/config.yaml -c yaml:///tmp/mp.conf
+cortx_setup.py config apply -f yaml://`pwd`/cluster.yaml -c yaml:///tmp/mp.conf
+```
+
+Run bootstrap command
+```bash
```
Comment (nitin-seagate) [inline test/setup/README.md:35]: Why do we need bash to execute python exec...

File: test/setup/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+utils_setup post_install --config yaml:/etc/cortx/cortx.conf --services message_bus
+csm_setup post_install --config yaml:/etc/cortx/cortx.conf --services agent
+motr_setup post_install --config yaml:/etc/cortx/cortx.conf --services fsm
+s3_setup post_install --config yaml:/etc/cortx/cortx.conf --services bg_producer
+```
+
+Run the following command  
+**Running The Tests**
+```bash
+./test_provisioner.py
```
Comment (nitin-seagate) [inline test/setup/README.md:53]: Why do we need bash to execute python exec...

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: test/setup/README.md (hunk 1 above)
Comment (tanujashinde0405) [reply test/setup/README.md]: done

File: test/setup/README.md (hunk 2 above)
Comment (tanujashinde0405) [reply test/setup/README.md]: done

File: test/setup/README.md (hunk 3 above)
Comment (tanujashinde0405) [reply test/setup/README.md:53]: done, replaced with python

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**